backend/app/models/assistants.py

```
@@ -31,7 +36,71 @@ class Assistant(AssistantBase, table=True):
     id: int = Field(default=None, primary_key=True)
     inserted_at: datetime = Field(default_factory=now, nullable=False)
     updated_at: datetime = Field(default_factory=now, nullable=False)
+    is_deleted: bool = Field(default=False, nullable=False)
+    deleted_at: Optional[datetime] = Field(default=None, nullable=True)
 
     # Relationships
     project: "Project" = Relationship(back_populates="assistants")
```

Ruff (0.12.2)

Undefined name `Project`

(F821)